api/build.gradle.kts

```diff
@@ -18,8 +19,8 @@ dependencies {
     testImplementation("org.springframework.boot:spring-boot-starter-test:2.2.5.RELEASE") {
         exclude(module = "junit")
     }
-    testImplementation("org.junit.jupiter:junit-jupiter-api:5.6.0")
-    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.6.0")
```

Comment:
What's wrong with the lines? As I know, It's a good practice to separate implementation and runtime dependencies

Reply:
You are right, but I can't start JUnit5 tests with separate dependencies. Other tests use JUnit4.

Reply:
Which ones use JUnit 4?

Reply:
GreetingRouterTest.java and StatusRouterTest.java, aren't they?

Reply:
If so it's my fault. I was planning to use JUnit 5

Reply:
I see, I mistakenly used junit4 TestRunner. I'll fix it soon. Thanks for pointing that out